documentation/modules/overview/con-configuration-points-connect.adoc

```diff
@@ -5,127 +5,257 @@
-* `FileStreamSinkConnector` streams data from Kafka to an external system, reading messages from a Kafka topic and creating a line for each in an output file.
---
+[discrete]
+== Connector plugin configuration
+
+Connector plugins provide the configuration for connecting to an external data source.
+Plugins contain a set of files that define the implementation required to connect to the data source.
+Connector plugins for many external systems are available for use with Kafka Connect.
+You can also create your own connector plugins.
 
-You can add other connectors using connector plugins, which are a set of JAR files or TGZ archives that define the implementation required to connect to certain types of external system.
+Connector plugins provide the configuration as a set of files that define the implementation required to connect to specific types of external data system.
+The configuration describes the source input data and target output data to feed into and out of Kafka Connect.
+The external source data must reference specific topics that will store the messages.
+The plugins might also contain the libraries and files needed to transform the data.
 
-You create a custom Kafka Connect image that uses new connector plugins.
+A Kafka Connect deployment can have one or more plugins, but only one version of each plugin.
 
-To create the image, you can use:
+You can create a custom Kafka Connect image that uses new connector plugins.
+You can create the image in two ways:
 
-* Kafka Connect configuration so that Strimzi creates the new image automatically.
-* A Kafka container image from {DockerRepository} as a base image.
+* link:{BookURLDeploying}#creating-new-image-using-kafka-connect-build-str[Automatically using Kafka Connect configuration^]
+* link:{BookURLDeploying}#creating-new-image-from-base-str[Manually using Dockerfile and a Kafka container image from {DockerRepository} as a base image^]
 
-For Strimzi to create the new image automatically, a `build` configuration requires `output` properties to reference a container registry that stores the container image,
-and `plugins` properties to list the connector plugins and their artifacts to add to the image.
+To create the container image automatically, you specify and deploy plugins with your Kafka Connect cluster using the `KafkaConnect` resource.
+Strimzi automatically downloads and adds the plugin artifacts to a new container image.
+The plugins are used by the workers that orchestrate the streaming processes.
 
-The `output` properties describe the type and name of the image, and optionally the name of the Secret containing the credentials needed to access the container registry.
-The `plugins` properties describe the type of artifact and the URL from which the artifact is downloaded. Additionally, you can specify a SHA-512 checksum to verify the artifact before unpacking it.
+Build (`build`) configuration automatically builds a container image with connector plugins.
```

Comment:
```suggestion
Build (`build`) configuration in the `KafkaConnect` resource automatically builds a container image with connector plugins.
```

Reply:
If you accept by suggestion above this sentence won't be needed.

Reply:
removed the sentence